shared/cfg/machines.cfg

```diff
@@ -46,3 +46,15 @@ variants:
         # Currently no USB support
         usbs =
         usb_devices =
+    - s390:
+        only s390
```

Comment:
```
    - s390x:
        only s390-virtio
        auto_cpu_model = "no"
        cpu_model = host
        machine_type = s390-ccw-virtio
        # No support for VGA yet
        vga = none
        inactivity_watcher = none
        take_regular_screendumps = no
        # Currently no USB support
        usbs =
        usb_devices =

```
After I change to only s390-virtio,  got an error as `No urls provided nor any arguments produced runable tests. Please double check the executed command.`

I am confusing... I think guest-hw.cfg should be same machines.cfg, but machines.cfg should same with which file? I am investigating.